pkg/sync/rpc/server.go

```diff
@@ -670,7 +670,7 @@ func (s *SyncAgentServer) BackupCreate(ctx context.Context, req *ptypes.BackupCr
 		return nil, err
 	}
 
-	backupID, replicaObj, err := backup.DoBackupCreate(&backup.CreateBackupParameters{
+	replicaObj, backupConfig, err := backup.DoBackupInit(&backup.CreateBackupParameters{
```

Comment:
NIT: Rename `replicaObj`. e.g., to `backupStatus` or `backupInfo`

Reply:
Updated. Thank you.